docs/setup/using_in_browser.md

```diff
@@ -0,0 +1,6 @@
+Initialization of library requires several properties of signer:
+
+- Is signer meets ethereum signer specification https://eips.ethereum.org/EIPS/eip-191
+- Public key. This is legacy parameter, which was used to initialized DID document
+
+These properties are not directly exposed by all signers and thus are determined in runtime by sending test sign request. In order to improve user expirience when library is used in browser it tries to get these parameters from local storage. Because library doesn't change storage application should set `isEthSigner` and `PublicKey` storage variables on login. Library doesn't send sign request if properties are not in local storage. Othewise user will have to sign twice if he was not logged in to cache server
```

Comment:
Does you have an example signer for which this isn't met?

Also, I also think markdown is a bit easier to maintain if each sentence is split into a separate line. This doesn't affect the rendered results but makes it easier to move between sentences and move them around.
```suggestion
These properties are not directly exposed by all signers and thus are determined in runtime by sending test sign request.
In order to improve user expirience when library is used in browser it tries to get these parameters from local storage.
Because library doesn't change storage application should set `isEthSigner` and `PublicKey` storage variables on login.
Library doesn't send sign request if properties are not in local storage. 
Otherwise user will have to sign twice if he was not logged in to cache server
```

Reply:
> Does you have an example signer for which this isn't met?

I'll give Metamask as an example